File: Beginner/fibonacci21.java
package basics;

public class fibonacci {

public static void main(String[] args) {

int n=5;
int f0=0;
int f1=1;
int ct=1;
while(ct<=n){
int f2=f1+f0;
f0=f1;
f1=f2;
ct++;
}
System.out.println(f0);
}

}

File: Beginner/fibonacci21.py
num=0
num1=1 
j=int(input("enter the number till where you want to see the fibonacci series"))
j=j-2
print(num)
print(num1)
for i in range(j):
    next1=num+num1
    print(next1)
    num=num1
    num1=next1
